tubemq-manager/src/main/java/org/apache/tubemq/manager/entry/NodeEntry.java

```diff
@@ -32,8 +32,8 @@
 @Data
 public class NodeEntry {
     @Id
-    @GeneratedValue(strategy= GenerationType.AUTO)
-    private long brokerId;
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
```

Comment:
what if clusters have conflicted broker ids. 
for example A cluster has broker which id is num_a, meanwhile B cluster also has broker with the same id.
It seems better that we need create a colume called brokerId without any annotation.